src/lib/components/tablev2/SingleSelectableContent.js

```diff
@@ -10,32 +10,50 @@ import {
   SortCaretWrapper,
   SortIncentive,
 } from './Tablestyle';
-import { TableContext } from './Tablev2.component';
+import { useTableContext } from './Tablev2.component';
+import { convertRemToPixels } from './TableUtil';
 
-export type SimpleContentProps = {
-  rowHeight: number,
-  outerTableHeight: number, //in pixel, the sum of height outside the table, pass it to virtualized table to calculate the height of the table
+export const tableRowHeight = {
+  // in rem unit
+  h32: '2.286',
+  h40: '2.858',
+  h48: '3.428',
+  h64: '4.572',
+};
+
+export type SingleSelectableContentProps = {
+  rowHeight: 'h32' | 'h40' | 'h48' | 'h64',
   separationLineVariant:
     | 'backgroundLevel1'
     | 'backgroundLevel2'
     | 'backgroundLevel3'
     | 'backgroundLevel4',
 };
 
-export default function SimpleContent({
+export default function SingleSelectableContent({
   rowHeight,
-  outerTableHeight,
   separationLineVariant,
-}: SimpleContentProps) {
-  const { headerGroups, getTableBodyProps, prepareRow, rows } = useContext(
-    TableContext,
-  );
+}: SingleSelectableContentProps) {
+  if (['h32', 'h40', 'h48', 'h64'].indexOf(rowHeight) === -1) {
+    console.error('Invalid table row height defined.');
+  }
+  if (
+    [
+      'backgroundLevel1',
+      'backgroundLevel2',
+      'backgroundLevel3',
+      'backgroundLevel4',
+    ].indexOf(separationLineVariant) === -1
+  ) {
+    console.error('Invalid table seperation line color defined.');
```

Comment:
I would reword the error message to `Invalid separationLineVariant props, expected backgroundLevel1, backgroundLevel2, backgroundLevel3 or backgroundLevel4 but received ${separationLineVariant}`

Reply:
Yes. indeed. it's more developer friendly ;)